### ISSUE TITLE
only drawable is supported for background on actionbar

### ISSUE THREAD

**Issue by daverix** [JakeWharton/ActionBarSherlock#247]
When setting my own background for my actionbar I can use a color value in ICS but not in previous versions:

values/themes.xml:

``` xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="Theme.XLTime" parent="@style/Theme.Sherlock.Light.DarkActionBar">
        <item name="actionBarStyle">@style/Widget.XLTime.ActionBar</item>
    </style>

    <style name="Widget.XLTime.ActionBar" parent="@style/Widget.Sherlock.ActionBar">
        <item name="background">#909fbc</item>
        <item name="backgroundStacked">#909fbc</item>
        <item name="backgroundSplit">#909fbc</item>
    </style>
</resources>
```

This works ok as it uses the android styles...

values-v14/themes.xml:

``` xml
<?xml version="1.0" encoding="utf-8"?>
<resources>
    <style name="Theme.XLTime" parent="@style/Theme.Sherlock.Light.DarkActionBar">
        <item name="android:actionBarStyle">@style/Widget.XLTime.ActionBar</item>
    </style>

    <style name="Widget.XLTime.ActionBar" parent="@android:style/Widget.Holo.ActionBar">
        <item name="android:background">#909fbc</item>
        <item name="android:backgroundStacked">#909fbc</item>
        <item name="android:backgroundSplit">#909fbc</item>
    </style>
</resources>
```

Using ABS 4.0 beta 6

**Comment by JakeWharton** [comment]
Pushed a possible fix to the `4.0-wip` branch. I can test it sometime later but if you want you can pull it in an give it a try now.

**Comment by JakeWharton** [comment]
Confirmed as working. Good find!
